CHANGELOG.md

```diff
@@ -54,6 +56,7 @@ https://docs.docker.com/engine/deprecated/ where you can find the target removal
 - Fix race in attachable network attachment [moby/moby#36191](https://github.com/moby/moby/pull/36191)
```

Comment:
Couple of lines up; we don't ship the diagnostic client, so can probably be left out (double check with @fcrisciani )